class.jetpack.php

```diff
@@ -2806,8 +2817,8 @@ public static function plugin_activation( $network_wide ) {
 	public static function do_version_bump( $version, $old_version ) {
 
 		if ( ! $old_version ) { // For new sites
-			// Setting up jetpack manage
-			Jetpack::activate_manage();
+			// There used to be stuff here, but this function might be useful to someone
```

Comment:
If we're removing the functionality, can we also remove the conditional? Should this be added to a deprecated list?